CHANGELOG.md

```diff
@@ -0,0 +1,42 @@
+v0.3.0-beta1
+============= 
+
+* using libdaemon to daemonize the ProxySQL process
+* added angel process that is able to monitor and restart ProxySQL
+* added separate error log
+* using MariaDB client library + libevent for connecting and querying backend MySQL servers
```

Comment:
libevent is used only for Monitoring module (maybe not a very important detail)
Also, it is worth noting that queries are re-executed if the backend goes away